src/commands/install-ndk.yml

```diff
@@ -24,12 +24,13 @@ steps:
 
         echo y | sdkmanager "ndk;<< parameters.version >>"
 
-        echo "export ANDROID_NDK_HOME=/opt/android/<< parameters.version >>" >> $BASH_ENV
-
-        if [[ -d $HOME/android-sdk/ndk/<< parameters.version >> ]] || [[ -d /opt/android/sdk/ndk/<< parameters.version >> ]]; then
+        if [[ -d $HOME/android-sdk/ndk/<< parameters.version >> ]]; then
+          echo "export ANDROID_NDK_HOME=$HOME/android-sdk/ndk/<< parameters.version >>" >> $BASH_ENV
+          echo "Android NDK installed"
```

Comment:
I'm thinking that instead of just saying the NDK was installed, what if we also showed its path:

```
android_ndk_home=""

if [[ -d $HOME/android-sdk/ndk/<< parameters.version >> ]]; then
  android_ndk_home="$HOME/android-sdk/ndk/<< parameters.version >>"

elif [[ -d /opt/android/sdk/ndk/<< parameters.version >> ]]; then
  android_ndk_home="/opt/android/<< parameters.version >>"

else
  echo "Android NDK did not install successfully."
  exit 1
fi

echo "export ANDROID_NDK_HOME=$android_ndk_home" >> "$BASH_ENV"
echo "Android NDK was installed at \"$android_ndk_home\"."
```